multiqc/modules/deeptools/estimateReadFiltering.py

```diff
@@ -0,0 +1,94 @@
+                self.deeptools_estimateReadFiltering[k] = v
+
+            if len(parsed_data) > 0:
+                self.add_data_source(f, section='estimateReadFiltering')
+
+        if len(self.deeptools_estimateReadFiltering) > 0:
+            header = OrderedDict()
+            header["M Entries"] = {'title': 'M entries', 'description': 'Number of entries in the file (millions)'}
+            header["% Aligned"] = {'title': '% Aligned', 'description': 'Percent of aligned entries'}
+            header["% Filtered"] = {'title': '% Tot. Filtered', 'description': 'Percent of alignment that would be filtered for any reason.'}
+            header["% Blacklisted"] = {'title': '% Blacklisted', 'description': 'Percent of alignments falling (at least partially) inside a blacklisted region'}
+            header["% MAPQ"] = {'title': '% MAPQ', 'description': 'Percent of alignments having MAPQ scores below the specified threshold'}
+            header["% Missing Flags"] = {'title': '% Missing Flags', 'description': 'Percent of alignments lacking at least on flag specified by --samFlagInclude'}
+            header["% Forbidden Flags"] = {'title': '% Forbidden Flags', 'description': 'Percent of alignments having at least one flag specified by --samFlagExclude'}
+            header["% deepTools Dupes"] = {'title': '% deepTools Dupes', 'description': 'Percent of alignments marked by deepTools as being duplicates'}
+            header["% Duplication"] = {'title': '% Duplication', 'description': 'Percent of alignments originally marked as being duplicates'}
+            header["% Singletons"] = {'title': '% Singletons', 'description': 'Percent of alignments that are singletons (i.e., paired-end reads where the mates don\'t align as a pair'}
+            header["% Strand Filtered"] = {'title': '% Strand Filtered', 'description': 'Percent of alignments arising from the wrong strand'}
+
+            d = dict()
+            for k, v in self.deeptools_estimateReadFiltering.items():
+                d[k] = {'M Entries': v['total'] / 1000000.0,
+                        '% Aligned': 100. * v['mapped'] / float(v['total']),
+                        '% Filtered': 100. * v['filtered'] / float(v['total']),
+                        '% Blacklisted': 100. * v['blacklisted'] / float(v['total']),
+                        '% Below MAPQ': 100. * v['mapq'] / float(v['total']),
+                        '% Missing Flags': 100. * v['required flags'] / float(v['total']),
+                        '% Forbidden Flags': 100. * v['excluded flags'] / float(v['total']),
+                        '% deepTools Dupes': 100. * v['internal dupes'] / float(v['total']),
+                        '% Duplication': 100. * v['dupes'] / float(v['total']),
+                        '% Singletons': 100. * v['singletons'] / float(v['total']),
+                        '% Strand Filtered': 100. * v['strand'] / float(v['total'])}
+
+            self.general_stats_addcols(d, header)
```

Comment:
These end up not summing to 100%, but to some larger indeterminate number. The values show numbers/percentages for each filtering setting, so many alignments will get double counted.

Reply:
Ah ok, that's a shame. In that case it may make sense to make a separate table it a deepTools section (`table.plot`). That way we keep the stats in a table but don't fill the General Stats table with loads of columns.